extras/liquibase-unlocker/pom.xml

```diff
@@ -36,14 +36,7 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-        </dependency>
```

Comment:
If I remember correctly instead of providing `commons-logging` we can use `jcl-over-slf4j`, right?